android-animation-arsenal/src/main/java/com/android_animation_arsenal/AnimationArsenal.java

```diff
@@ -62,15 +58,14 @@ public static Transition getExplodeTransition(Transition.TransitionListener list
     public static Transition getSlideTransition(Transition.TransitionListener listener, int
             duration, int gravity)
     {
-        if(Build.VERSION.SDK_INT < Build.VERSION_CODES.LOLLIPOP)
+        if(Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
```

Comment:
I think that should be a check the library has to do to make sure everything is running as expected..